test/controllers/drivers_controller_test.rb

```diff
@@ -55,121 +51,183 @@
   describe "create" do
     it "can create a new driver with valid information accurately, and redirect" do
       # Arrange
-      # Set up the form data
-
+      driver_hash = {
+        driver: {
+          name: "John Do",
+            vin: "ww234444112345651",
+            available: true
+        },
+      }
       # Act-Assert
       # Ensure that there is a change of 1 in Driver.count
+      expect {
+        post drivers_path, params: driver_hash
+      }.must_change "Driver.count", 1
 
       # Assert
       # Find the newly created Driver, and check that all its attributes match what was given in the form data
+      new_driver = Driver.find_by(name: driver_hash[:driver][:name])
+      expect(new_driver.vin).must_equal driver_hash[:driver][:vin]
+      expect(new_driver.available).must_equal driver_hash[:driver][:available]
+
       # Check that the controller redirected the user
+      must_respond_with :redirect
+      must_redirect_to driver_path(new_driver.id)
 
     end
 
-    it "does not create a driver if the form data violates Driver validations, and responds with a redirect" do
+    it "does not create a driver if the form data violates Driver validations" do
       # Note: This will not pass until ActiveRecord Validations lesson
       # Arrange
       # Set up the form data so that it violates Driver validations
+      driver_hash = {
+        driver: {
+          name: "John Do",
+            vin: "ww234",
+            available: false
+        },
+      }
 
       # Act-Assert
       # Ensure that there is no change in Driver.count
+      expect {
+        post drivers_path, params: driver_hash
+      }.wont_change "Driver.count"
 
-      # Assert
-      # Check that the controller redirects
+      assert_template :new
```

Comment:
Better to just check the status code.  

```suggestion
      must_respond_with :bad_request
```